### PR TITLE
Add Database Tools

### PR DESCRIPTION
**Pull request by mgramin**
https://github.com/mgramin/awesome-db-tools

This is a community driven list of database tools (ide, cli, managing, monitoring, migrations, modellers, visualization etc)

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of sub-lists, preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Hey, sorry for the lack of response, please see #1363.

**Comment by DovAmir** [comment]
This is a PR review

1) unnececary ' character

> PGDeltaStream - A Golang webserver to stream Postgres changes atleast-once over websockets, using Postgres' logical decoding feature.

2) no period at end

> OmniDB - Web tool for database management

3) no license aggrement section

please review
https://github.com/sindresorhus/awesome/pull/1331
thanks

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

**Comment by sindresorhus** [comment]
I think the list should be renamed to `Awesome Database Tools`. No good reason to shorten it to `DB`.

**Comment by sindresorhus** [comment]
> Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
* Should be placed on the right side of the readme heading.

**Comment by sindresorhus** [comment]
Heading `awesome-db-tools` => `Awesome DB Tools`

**Comment by sindresorhus** [comment]
> * **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)

**Comment by mgramin** [comment]
@sindresorhus fixed

**Comment by sindresorhus** [comment]
The list needs a better into than `Here we will collect information about various db-tools such as db-IDE, db-managment tools, db-migration tools, db-metric tools, db-ops tools etc etc.`.

File: readme.md [hunk 1]
```diff
@@ -395,6 +395,7 @@
 - [CouchDB](https://github.com/quangv/awesome-couchdb#readme) - Document-oriented NoSQL database.
 - [HBase](https://github.com/rayokota/awesome-hbase#readme) - Distributed, scalable, big data store.
 - [NoSQL Guides](https://github.com/erictleung/awesome-nosql-guides#readme) - Help on using non-relational, distributed, open-source, and horizontally scalable databases.
+- [Database tools](https://github.com/mgramin/awesome-db-tools) - List of database tools (IDE, CLI, managing, monitoring, migrations, modelers, visualization etc).
```
Comment (sindresorhus) [inline readme.md]: The title should be titleized.

The word `List` is implied. All Awesome lists are lists...

`database tools` is just repeating the the tile.

Please write a better description.

Comment (sindresorhus) [inline readme.md]:
```suggestion
- [Database tools](https://github.com/mgramin/awesome-db-tools#readme) - List of database tools (IDE, CLI, managing, monitoring, migrations, modelers, visualization etc).
```

**Comment by mgramin** [comment]
@sindresorhus fixed

**Comment by sindresorhus** [comment]
You did not address all my feedback.

I also think many of the descriptions for items could be better written. Please go through them and try to improve them.

For example, `Command line util for export oracle schema to set of ddl scripts.` is not a good sentence. Neither is `Is a open-source, ultra-lightweight data access layer for HTTP clients.`.

**Pull request closed without merging**